api-docs/list-clients.md

```diff
@@ -2,7 +2,8 @@
 
 API is provided to query a list of clients from the database.
 
-##Request
+## Request
```

Comment:
Github markdown requires a space character between the `##` and the text in order for it to show up as a header. Compare [before](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/api-docs/list-clients.md) and [after](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/b4f8c7fb3f5477bedf2a04a3801e5ae57fcce72d/api-docs/list-clients.md)

Reply:
Good to know